packages/components/check-tag/src/check-tag.vue

```diff
@@ -17,7 +17,11 @@ const props = defineProps(checkTagProps)
 const emit = defineEmits(checkTagEmits)
 
 const ns = useNamespace('check-tag')
-const containerKls = computed(() => [ns.b(), ns.is('checked', props.checked)])
+const containerKls = computed(() => [
+  ns.b(),
+  ns.is('checked', props.checked),
+  ns.m(props.type || 'primary'),
```

Comment:
`props.type` has a default value, maybe we don't need to set the default value repeatedly?

Reply:
It's for setting empty string to `type`.

Reply:
`type` does not seem to allow accepting empty strings.

Reply:
Yes, It's an insurance operation. If someone set `null`, we need to use the default value.